src/journal/journald-server.c

```diff
@@ -818,6 +818,78 @@ static bool shall_try_append_again(JournalFile *f, int r) {
         }
 }
 
+typedef struct FailedEntry {
+        int r;
+        size_t nb_entries;
+        size_t nb_items;
+        size_t total_size;
+        const char *file;
+        const char *message;
+        RateLimit rl;
```

Comment:
We don't usually abbreviate in structs so I'd write `ratelimit` out in full as the variable name here